pkg/core/fee/opcode.go

```diff
@@ -20,6 +20,8 @@ var coefficients = [256]uint16{
 	opcode.PUSHINT64:    1 << 0,
 	opcode.PUSHINT128:   1 << 2,
 	opcode.PUSHINT256:   1 << 2,
+	opcode.PUSHT:        1 << 0,
+	opcode.PUSHF:        1 << 0,
```

Comment:
To be checked again after the respective core change.

Reply:
They are the same, see https://github.com/neo-project/neo/pull/2832/files/66d5cb1a36a2d3b9c2b9496a363a694aa18dae38..fa6ab8f33076dc84448723365bef55b71503a4f9